KD_Lib/models/nin.py

```diff
@@ -0,0 +1,50 @@
+from torch import nn
+
+
+class NIN(nn.Module):
+    def __init__(self, num_classes=10, in_channels=3):
```

Comment:
maybe we can give a more meaningful name to this.

Reply:
Ok, will rename it. :+1: